book/website/community-handbook/fireside-chat/fireside-chat-roles.md

```diff
@@ -0,0 +1,90 @@
+* Host 30 mins calls with the speakers before the event to identify what they would like to speak about.
+* Frame questions based on their discussions with the speakers, and invite speakers to help finalise the question (using Slack and the planning document).
+* Considering the flow of the discussion, decide on the order of the questions that will be asked.
+  * Each co-host should have equal opportunity to get the ‘stage time’
+  * They should agree on which questions they will pose to guide the discussion.
+  * They should pose 1 question to each speaker who takes a lead on responding. 1-2 follow-up responses can be invited from other speakers.
+    * Make sure that the speakers know in what order they will be speaking, and what questions will be asked.
+    * Coordinate via the Slack channel, and share any notes about the format via the shared document.
+  * They should pose at least one question to each other - either directly, or as a follow-up to the main question.
+* Finalise a question for the closing statement (30-60 seconds take away from each speaker)
+
+(ch-fireside-chat-roles-speakers)=
+## Speakers - Guide the Discussion
+
+We host 5-6 speakers in the panel, which includes the facilitators.
+It is important to note that fireside Chats don't have the traditional role of facilitators versus speakers.
+Facilitators are invited based on their experience on the topic and they also respond to questions, adding to the discussion from their experiences.
+
+* Once agreed upon, all speakers will be invited to join a private Slack channel to participate in asynchronous planning discussions.
+* They will supply a short bio and image to be shared on the Eventbrite page.
+* They will be invited to attend a call (30) minutes before the event to discuss the topic and help the co-hosts identify what specific area of the topic they will talk about.
+* They will be coordinating with the co-host to finalise the specific question posed to them, and if they would like, they can suggest another question for the other panellists.
+* They will have access to the internal planning documents where they can find all information they need, as well as have opportunities to give feedback by directly collaborating on the Google document.
+* They should check what in addition to questions posed to them, questions they would like to respond to as a follow-up discussion.
+* Join Zoom at least 5-10 minutes before the event to test their system (microphone, camera, internet, and access to document used for the session)
+* It is important that all speakers get the same ‘stage time’ -- so please take this space to share your thoughts if you are generally not the one speaking out, and if you generally tend to speak more, please intentionally create space for others to speak.
```

Comment:
From here down the person changes from "they will" (third person) to "you" (second person) - not sure which one we prefer to use